R/new_stan_model.R

```diff
@@ -51,6 +54,7 @@
 new_stan_model <- function(
   parameters,
   parameter_definitions,
+  variable_definitions = NULL,
```

Comment:
It's useful to be able to define variables in the Stan model. For the busulfan model it was needed to calculate BMI and FFM on-the-fly.

Reply:
makes sense to me. since this is a new feature we should add a unit test?